src/__tests__/Button.test.js

```diff
@@ -20,14 +20,6 @@ describe('rowComp(Button)', () => {
 });
 
 describe('Pure <Button>', () => {
-    it('renders a <button type=button>', () => {
```

Comment:
Use `xit()` to disabled this test case temporarily? May restore it after Safari 11 release XD